packages/apputils/src/sanitizer.ts

```diff
@@ -25,7 +25,7 @@ class Sanitizer implements ISanitizer {
 
   private _options: sanitize.IOptions = {
     allowedTags: sanitize.defaults.allowedTags
-      .concat('h1', 'h2', 'img', 'span', 'audio', 'video'),
+      .concat('h1', 'h2', 'img', 'span', 'audio', 'video', 'mark'),
```

Comment:
Is it possible to have a separate sanitizer list for the completer context where mark is just about all we would expect?

Reply:
Good call, done.